docs/jade-getting-started.md

```diff
@@ -289,17 +292,21 @@ to build `jade-data-repo`.
 
 * You will need to run `render-configs.sh` before running integration tests.
 
-* Certain environment variables need to be set beforehand. Instances of `zzz`
-should be replaced by your initials or the environment (i.e. `dev`):
+Certain environment variables need to be set beforehand. Instances of `zzz`
+should be replaced by your initials:
 
 ```
 export VAULT_ADDR=https://clotho.broadinstitute.org:8200
 export PROXY_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
 export GOOGLE_CLOUD_PROJECT=broad-jade-zzz
 export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
 export GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
-export GOOGLE_CLOUD_DATA_PROJECT=broad-jade-zzz-data
+export GOOGLE_CLOUD_DATA_PROJECT=broad-jade-dev-data
 export IT_JADE_API_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
+
+export DB_MIGRATE_DROPALLONSTART=true
+export GOOGLE_ALLOWREUSEEXISTINGBUCKETS=true
+export GOOGLE_ALLOWREUSEEXISTINGPROJECTS=true
 ```
```

Comment:
I snuck in a PR that updated these environment variables, so you'll probably just need to rebase! https://github.com/DataBiosphere/jade-data-repo/pull/867